tests/queries/0_stateless/01164_detach_attach_partition_race.sh

```diff
@@ -13,7 +16,9 @@ $CLICKHOUSE_CLIENT -q "insert into mt values (3)"
 function thread_insert()
 {
     while true; do
-        $CLICKHOUSE_CLIENT -q "insert into mt values (rand())";
+        # It might be the case that the threads are terminated and exited, but some children didn't and they are still sending queries when we are dropping tables.
+        # That's why the "Table doesn't exist" error is allowed, while other errors don't.
+        $CLICKHOUSE_CLIENT -q "insert into mt values (rand())" 2>&1 | tr -d '\n' | rg -v "Table .+ doesn't exist";
```

Comment:
instead of ignoring errors, what do you think about doing the same thing I did for some other stateless tests with `timeout`
https://github.com/ClickHouse/ClickHouse/pull/51774/files

Reply:
Good idea, let's do it.